### ISSUE TITLE
Config/Settings Area

### ISSUE THREAD

**Issue by canstudios-louisem** [adaptlearning/adapt_authoring#1702]
There is currently no central config/Settings area in the AT.

I don't want to replace the `config.json` and I think if we were to create one that the `config.json` should override any configs plus there are things like DB details that will always have to be in a file somewhere.

But the ability to set things like SMTP details and view what the current config would be really useful.

I'm not sure if we would want to just update the config or store the details in the db.

**Comment by taylortom** [comment]
This ties nicely into the impending localisation feature. I'm in two minds as to whether we should override config.json or not. I can imagine it'd be useful to do so via the interface...

**Comment by canstudios-louisem** [comment]
Yeah I'm not sure if the config should be used to override this or this override the config. There are benefits to both.

**Comment by taylortom** [comment]
I've done something similar to this here: [adapt-sysadmin](https://github.com/taylortom/adapt-sysadmin). It acts as a central page for all sys-admin-type features, and allows other child-plugins to be added:
- [adapt-sysadmin-tags](https://github.com/taylortom/adapt-sysadmin-tags)
- [adapt-sysadmin-serverlog](https://github.com/taylortom/adapt-sysadmin-serverlog)
- [adapt-sysadmin-serverconfig](https://github.com/taylortom/adapt-sysadmin-serverconfig)

![screen shot 2018-04-12 at 17 05 01](https://user-images.githubusercontent.com/1059083/38689477-b303a7ba-3e73-11e8-9178-b881ba51cd74.png)
